subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -100,7 +100,7 @@
                                             <li><a href="/userguide/web_project_tutorial.html">Web Application Quickstart</a></li>
                                             <li><a href="https://guides.gradle.org/writing-gradle-tasks/">Writing Custom Script Tasks</a></li>
                                             <li><a href="/userguide/custom_tasks.html">Writing Custom Task Classes</a></li>
-                                            <li><a href="/userguide/artifact_dependencies_tutorial.html">Dependency Management for Java Projects</a></li>
+                                            <li><a href="/userguide/dependency_management_java_projects.html">Dependency Management for Java Projects</a></li>
```

Comment:
This isn't a tutorial, so I think belongs on one level up here.

Reply:
It probably never should have been in the "tutorials" section.

Reply:
Also this link doesn't work... we should fix that ;)

Reply:
done. Also moved it out of tutorials.

Regarding "tutorials" I don't think we have a good definition _what_ a tutorial is right now. At least I don't know it. Is that what we used to call "Getting started guide"?

Reply:
My definition of a tutorial is: "Creating something through a discrete set of instructions". So, yeah, basically a "getting started guide", though one could start with a pre-baked project (as Grails guides often do).